Drop after https://reviews.freebsd.org/D31003 lands

meson.build:1521:4: ERROR: Invalid version of dependency, need 'libdrm_amdgpu' ['>=2.4.107'] found '2.4.106'.

../src/amd/vulkan/winsys/amdgpu/radv_amdgpu_bo.c:427:64: error: use of undeclared identifier 'AMDGPU_VA_RANGE_REPLAYABLE'
                             (flags & RADEON_FLAG_REPLAYABLE ? AMDGPU_VA_RANGE_REPLAYABLE : 0);
                                                               ^

--- meson.build.orig	2021-04-14 06:35:30 UTC
+++ meson.build
@@ -1452,7 +1452,7 @@ dep_libdrm_radeon = null_dep
 dep_libdrm_nouveau = null_dep
 dep_libdrm_intel = null_dep
 
-_drm_amdgpu_ver = '2.4.107'
+_drm_amdgpu_ver = '2.4.105'
 _drm_radeon_ver = '2.4.71'
 _drm_nouveau_ver = '2.4.102'
 _drm_intel_ver = '2.4.75'
--- src/amd/vulkan/winsys/amdgpu/radv_amdgpu_bo.c.orig	2021-07-05 19:49:32 UTC
+++ src/amd/vulkan/winsys/amdgpu/radv_amdgpu_bo.c
@@ -424,6 +424,9 @@ radv_amdgpu_winsys_bo_create(struct radeon_winsys *_ws
 
    const uint64_t va_flags = AMDGPU_VA_RANGE_HIGH |
                              (flags & RADEON_FLAG_32BIT ? AMDGPU_VA_RANGE_32_BIT : 0) |
+#ifndef AMDGPU_VA_RANGE_REPLAYABLE
+#define AMDGPU_VA_RANGE_REPLAYABLE     0x4
+#endif
                              (flags & RADEON_FLAG_REPLAYABLE ? AMDGPU_VA_RANGE_REPLAYABLE : 0);
    r = amdgpu_va_range_alloc(ws->dev, amdgpu_gpu_va_range_general, size, virt_alignment, replay_address,
                              &va, &va_handle, va_flags);
